docs/cli.md

```diff
@@ -23,28 +23,9 @@ The NEAR AI CLI allows you to interact with NEAR AI's services to [create agents
     ``` bash
     git clone git@github.com:nearai/nearai.git
     cd nearai
-    pip install -e .
+    ./install.sh
     ```
 
-??? abstract "Python Version"
```

Comment:
maybe instead of removing, we can add a tab for `uv`?

Reply:
ah sure, was just simplifying this